packages/auth/src/accessToken/routes.ts

```diff
@@ -90,34 +100,47 @@ async function revokeToken(
   deps: ServiceDependencies,
   ctx: RevokeContext
 ): Promise<void> {
-  const token = (ctx.headers['authorization'] ?? '').replace('GNAP ', '')
-  const { id: managementId } = ctx.params
-  await deps.accessTokenService.revoke(managementId, token)
+  await deps.accessTokenService.revoke({
+    managementId: ctx.accessToken.managementId,
+    tokenValue: ctx.accessToken.value
+  })
+
```

Comment:
Do we want to actually check if we weren't able to revoke the token here? I'm guessing, like the old behaviour we should continue returning 204 since it could have been revoked previously for example

Reply:
We only would not have been able to revoke the token if we couldn't find it, right? In that case, it either never existed or was revoked a while ago and cleaned up. I think you are right, we should always return the 204 and I don't think what value it adds saying "token was revoked before / never existed"